src/DssSpell.sol

```diff
@@ -51,91 +44,101 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
     // uint256 internal constant X_PCT_RATE = ;
-    uint256 internal constant FIVE_PT_FIVE_FOUR_PCT_RATE  = 1000000001709786974743980088;
-    uint256 internal constant FIVE_PT_SEVEN_NINE_PCT_RATE = 1000000001784811360376128985;
-    uint256 internal constant SIX_PT_TWO_NINE_PCT_RATE    = 1000000001934329706253075715;
-
+    
     // ---------- Math ----------
-    uint256 internal constant THOUSAND = 10 ** 3;
-    uint256 internal constant MILLION  = 10 ** 6;
-    uint256 internal constant BILLION  = 10 ** 9;
-    uint256 internal constant WAD      = 10 ** 18;
-    uint256 internal constant RAD      = 10 ** 45;
+    uint256 constant internal MILLION  = 10 **  6;
 
-    // ---------- SBE parameter changes ----------
-    address internal immutable MCD_VOW            = DssExecLib.vow();
-    address internal immutable MCD_FLAP           = DssExecLib.flap();
+    address immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
 
-    // ---------- Reduce PSM-GUSD-A Debt Ceiling ----------
-    VatAbstract internal immutable vat            = VatAbstract(DssExecLib.vat());
+    // Function from https://github.com/makerdao/spells-goerli/blob/7d783931a6799fe8278e416b5ac60d4bb9c20047/archive/2022-11-14-DssSpell/Goerli-DssSpell.sol#L59
```

Comment:
this comment is missing Note prefix

Reply:
This is not an action, just a comment about the source of the function we are using.  No `Note` prefix is used in previous spells either. Are you sure we need to add it?

Reply:
> No Note prefix is used in previous spells either.

the rule to use "Note" introduced recently (https://github.com/makerdao/pe-checklists/commit/bc2c4fd9b75ef5bfbcc595bbddda4aec1e673578#diff-559957044fa021df9dab43dc9b2a87b7640b9d7374ee18ccf369c8053a59e4f6), so it maybe for this reason. 

The general idea behind it is to separate the comments from an instruction. In this case it's a note to another developer, so it would be nice to prefix it with `Note:`. 

But as it's not critical I'd leave it up to you! from my side this should be nit. Sorry for missing this :(